src/OneDriveSdk.WinStore/Extensions/OneDriveClientExtensions.cs

```diff
@@ -115,10 +115,11 @@ public static class OneDriveClientExtensions
         public static IOneDriveClient GetClientUsingOnlineIdAuthenticator(
             string[] scopes,
             string returnUrl = null,
-            IHttpProvider httpProvider = null)
+            IHttpProvider httpProvider = null,
+            CredentialPromptType promptType = CredentialPromptType.PromptIfNeeded)
         {
             return new OneDriveClient(
-                new AppConfig { MicrosoftAccountScopes = scopes },
+                new AppConfig { MicrosoftAccountScopes = scopes, MicrosoftAccountPromptType = promptType },
```

Comment:
Update method documentation header to include promptType. Same comment for GetUniversalClient.